src/Components/BookCard/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from "react";
+import "./bookcard.css";
+
+function BookCard({ bookCover, title, author, genre }) {
+  return (
+    <div>
+      <img src={bookCover} alt={title + " cover"} />
+      <h2>{title}</h2>
+      <p>{author}</p>
+      <p>{genre}</p>
+    </div>
+  );
```

Comment:
lose semi-colon

Reply:
Mmm we can discuss stylistic choices tomorrow perhaps?